runtime.md

```diff
@@ -56,10 +56,15 @@ The pre-start hooks are called after the container process is spawned, but befor
 They are called after the container namespaces are created on Linux, so they provide an opportunity to customize the container.
 In Linux, for e.g., the network namespace could be configured in this hook.
 
+The hooks MUST be called in the listed order (`Prestart[0]` first, `Prestart[1]` second, …).
```

Comment:
The JSON key is `prestart`, not `Prestart`.  And @vbatts [feels like the parenthetical may be too much][1] (at least for #142).

[1]: https://github.com/opencontainers/specs/pull/142/files#r38784041